packages/components/src/ConditionalFilter/GroupFilter.tsx

```diff
@@ -196,7 +235,7 @@ const Group = ({
                     {showMoreTitle}
                   </MenuItem>
                 ) : (
-                  <span hidden value=""></span>
+                  <span hidden></span>
```

Comment:
The value has to be there (PF issues). If you are having issues with TS, you can disable it for this line.